### PR TITLE
Add new tests

### PR DESCRIPTION
**Pull request by b-per**
This is a:
- [x] bug fix PR with no breaking changes
- [ ] new functionality

## Description & motivation

Modify the CI process by adding a new project to handle use cases that we can't handle with the first one:
- no exposure
- no metric
- no overwrite of the default seed
- (we can add more later)

I have also updated the CI steps to run the second project after the first one

This is expected to fail now because of #193 but if the issue is the same as the one linked we should merge the code and then make sure that the fix for the issue above passes the CI tests.

### CODE REVIEW

**Comment by b-per** [comment]
The error is now the same as the reported one, so I think it would be OK to merge this (currently failing) PR:
```
12:40:26  Database Error in model fct_source_directories (models/marts/structure/fct_source_directories.sql)
12:40:26    invalid input syntax for integer: "fct_source_directories"
12:40:26    LINE 8: where fct_name = 'fct_source_directories
```

File: integration_tests_2/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Test dbt Project
+
+The models within this folder are used for testing use cases that can't be checked as part of the other `integration_tests` project. No tests are defined as the main purpose is to ensure that all models run properly in specific situations.
```
Comment (graciegoheen) [inline integration_tests_2/README.md:3]: Can we define what the edge cases are that this is capturing?

- no metrics
- no exposures
- no exceptions seed

**Review by dave-connors-3 — APPROVED** [review]
🦞
